src/wp-settings.php

```diff
@@ -330,6 +330,7 @@
 require ABSPATH . WPINC . '/sitemaps/providers/class-wp-sitemaps-taxonomies.php';
 require ABSPATH . WPINC . '/sitemaps/providers/class-wp-sitemaps-users.php';
 require ABSPATH . WPINC . '/class-wp-block-bindings-registry.php';
+require ABSPATH . WPINC . '/class-wp-block-bindings-source.php';
```

Comment:
Nit: it's probably safer to define first the standalone class `WP_Block_Bindings_Source` as it gets referenced in `WP_Block_Bindings_Registry`: 
```suggestion
require ABSPATH . WPINC . '/class-wp-block-bindings-source.php';
require ABSPATH . WPINC . '/class-wp-block-bindings-registry.php';
```

Reply:
Gotcha, thanks!

Reply:
Done in 5f36c82bb33513b1620f294c793623c23dfdce82